pkgs/development/compilers/jetbrains-jdk/jcef.nix

```diff
@@ -73,6 +73,25 @@ let rpath = lib.makeLibraryPath [
 ];
 
 buildType = if debugBuild then "Debug" else "Release";
+lookupTable = {
+  "x86_64-linux" = {
+    clang_format_version = "dd736afb28430c9782750fc0fd5f0ed497399263";
```

Comment:
It looks like these  (clang_format_*) don't vary by system so didn't need moved.